.stylelintrc.js

```diff
@@ -57,6 +57,9 @@ module.exports = {
           'ActiveText',
           'Highlight',
           'ButtonText',
+          'SelectedItem',
+          'SelectedItemText',
+          'GrayText',
         ],
```

Comment:
Nitpick - should we Alpha sort these?

Reply:
👍 I’ll update to [match our stylelint config](https://github.com/wagtail/stylelint-config-wagtail/blob/9eef3b7442c5ee143838a85d6137e393706d9ca0/index.js#L75), which has all values, ordered according to their order in the spec.